packages/shadydom/src/patch-events.js

```diff
@@ -271,6 +272,31 @@ let EventPatches = {
   },
 };
 
+// Some browsers have missing or broken descriptors. If this is the case
+// eventPhase can't be polyfilled. For example, Chrome 41 does not
+// have prototypical descriptors on DOM objects and Safari 9 has broken ones
+// that cannot be used.
+const eventPhaseDescriptor =
+  utils.settings.hasDescriptors &&
+  Object.getOwnPropertyDescriptor(Event.prototype, 'eventPhase');
+if (eventPhaseDescriptor) {
+  Object.defineProperty(EventPatches, 'eventPhase', {
+    get() {
+      return this.currentTarget === this.target
```

Comment:
Events currently aren't patched to have `SHADY_PREFIX` properties. The patches for elements and events are treated differently.

Element prototypes are patched and in noPatch mode, they aren't patched. We access the `SHADY_PREFIX` properties because they always exist and if a noPatch wrapper calls a method, we can safely call the `SHADY_PREFIX` versions.

Event instances are patched so we will only ever get to this code on a patched instance. I can't say for sure why the Event prototype isn't patched and it's something perhaps we can revisit, but this is how it works right now.